frontend/database/107_add_submissions_school_id.sql

```diff
@@ -0,0 +1,18 @@
+-- Add school_id to Submissions
+ALTER TABLE `Submissions`
+ADD COLUMN `school_id` int(11) DEFAULT NULL,
+ADD KEY `school_id` (`school_id`),
+ADD CONSTRAINT `fk_sc_school_id` FOREIGN KEY (`school_id`) REFERENCES `Schools`(`school_id`) ON DELETE NO ACTION ON UPDATE NO ACTION;
```

Comment:
```suggestion
ADD CONSTRAINT `fk_ss_school_id` FOREIGN KEY (`school_id`) REFERENCES `Schools`(`school_id`) ON DELETE NO ACTION ON UPDATE NO ACTION;
```

nit: las llaves foráneas llevan la primer letra de las tablas relacionadas.